### ISSUE TITLE
[Feature Request] Add custom headers option to the settings

### ISSUE THREAD

**Issue by PROPHESSOR** [balisujohn/localwriter#15]
It's useful when you want to use ChatGPT for example or if your API uses some kind of authorization

**Comment by Asherathe** [comment]
Yes, even people using an LLM locally may have access secured with an API key to prevent random web pages from spying on their text generation.

**Comment by kbradsha** [comment]
I see information in the documentation referencing an API key in the localwriter.json but it never seems to work.
All I get is 401 Unauthorized when attempting to use it.  I know the API key I'm using is correct.  It's only 3 characters long.
